README.md

```diff
@@ -1,48 +1,84 @@
+This is simple react web application you can see companies information and also see the particular company full details.
 
+## Built With
+
+- HTML
+- CSS
+- React JS
+- React Router Dom
+- Redux Toolkit
+
+## Screen Shots
+
+### Home Page
+
+![home-page](./src/assets/screen%20shots/home-page.png)
+
+### Details Page
+
+![details-page](./src/assets/screen%20shots/details-page.png)
+
+## Live Preview
+
+[Live Preview](https://companies-information.netlify.app)
+
+## Vide Documentation
+
+[Watch Video](https://drive.google.com/file/d/18I0MSA-wx-_5fPwyKz18UPAPDGZIEF-c/view?usp=sharing)
 
 ## Getting Started
 
-This repository includes files with plain SQL that can be used to recreate a database:
+To get a local copy of this exercise, Please follow these simple example steps.
 
-- Use [schema.sql](./schema.sql) to create all tables.
-- Use [data.sql](./data.sql) to populate tables with sample data.
-- Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database. **Important note: this file might include queries that make changes in the database (e.g., remove records). Use them responsibly!**
+1. Clone this repository or download the Zip folder:
+   `https://github.com/aamirkhan2478/companies-details-react.git`
+2. Navigate to the location of the folder in your machine:
+   `you@your-Pc-name:~$ cd <folder>`
+3. Press Enter to navigate to your local clone.
+   Now you can go and play with it as you wish :smile:
 
+### Run this command to install dependencies
 
-## Authors
+> npm install
 
-👤 **Author1**
+### Run this command to run server
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+> npm start
 
-👤 **Author2**
+### Run this command to build your webpack
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+> npm run build
 
-## 🤝 Contributing
+### Run this command to test our project
 
-Contributions, issues, and feature requests are welcome!
+> npm test
+
+## Authors
 
-Feel free to check the [issues page](../../issues/).
+:bust_in_silhouette: **Aamir Khan**
+
+- GitHub: [@githubhandle](https://github.com/aamirkhan2478)
+- Twitter: [@twitterhandle](https://twitter.com/SmartKhan2478)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/aamir-khan-302a44237/)
 
 ## Show your support
 
-Give a ⭐️ if you like this project!
+Give a :star:️ if you like this project!
+
+## :handshake: Contributing
+
+Contributions, issues, and feature requests are welcome!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- Hat tip to anyone whose code was used :beginner:
+- Inspiration :cupid:
+- Microverse program :zap:
+- My standup team :bow_and_arrow:
+- My family's support :raised_hands:
```

Comment:
- [ ] Please give appropriate credit to the author [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) in the README file for the design as requested in the project requirements.

![image](https://user-images.githubusercontent.com/83803414/194341162-9f3f213d-2d48-471c-86bc-2a6e1970adfb.png)